common/src/leap/soledad/common/tests/util.py

```diff
@@ -362,8 +367,14 @@ def start(self):
 %s""" % (
                     self.process.returncode, got_stdout, got_stderr))
             time.sleep(0.01)
+            if (time.time() - boot_time) > self.BOOT_TIMEOUT_SECONDS:
+                self.stop()
+                raise Exception("Timeout starting couch")
```

Comment:
PS: I know that on a perfect execution the couchdb will create log file, but this check was intended for the not-best-case. :P